server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
+De vlag wapperde in de wind.
+Ben je rechts- of linkshandig?
+Hij klikte op de snelkoppeling.
+Ze had besloten dat ze nog even de kat uit de boom zou kijken.
+Zijn afwachtende houding ergerde mij.
+Het internet lijkt niet meer te werken.
+Wat een stomme uitdrukking is dat.
+Dat kan helemaal niet.
+De rits blokkeerde halfweg.
+Ik had koffie geknoeid op mijn muismat.
+Er was duidelijk sprake van verzachtende omstandigheden.
+Ze reden samen op de tandem.
+Er valt niets meer te redden.
+Snel smeet ik een reddingsboei in haar richting.
+Ik vind drop heerlijk, maar sommige mensen houden niet van de anijssmaak.
+Toen ik klein was heb ik een keer drie maanden in het ziekenhuis gelegen.
+Zij is zeer intelligent.
+Hij kan goed luisteren.
+In de klas kon ik mij nooit concentreren.
+Als psycholoog moet je zeer nauwlettend luisteren want ieder detail kan belangrijk zijn.
+Mijn broek zakt af.
+In het weekend ga ik soms gokken in het casino.
+We zijn gaan zeilen bij de Waddeneilanden.
+Wist jij dat Reykjavik de hoofdstad van IJsland is?
+Mijn Noors is niet meer zo goed als het ooit geweest is.
+Ik heb op een boorplatform gewerkt toen ik jong was.
+De zee kon soms ruw zijn.
+Ik ga een maand rondreizen in India.
+Vanuit Indonesië zijn we via Singapore en Maleisië naar Thailand gegaan.
+Sommige eilanden in de Chinese Zee worden betwist.
+In de jaren tachtig was Japan het groeiland bij uitstek.
+Je moet het soms niet te ver gaan zoeken.
+Er zijn prachtige reisbestemmingen in Duitsland.
+Een groot deel van Oostenrijk is eigenlijk vrij vlak.
+Zwitserland ligt bijna volledig in de bergen.
+De treinrit richting Bern was genieten.
+Oehoe is een onomatopee.
+Hij sloeg de inbreker met een deegrol.
+Je mag dat niet zomaar veralgemenen.
+Ik denk dat je mening vooral gebaseerd is op vooroordelen.
+De keuze was vooral pragmatisch.
+Hij had het pakketje verstopt in de mannentoiletten.
+De schoonmaakploeg was net gepasseerd in de vrouwentoiletten.
+Ze zochten naar een blanke man van circa veertig jaar van één meter tweeënzeventig.
+De generator had de geest gegeven.
+De stroompanne duurde bijna twee uur.
+Er is veel tijd gekropen in de voorbereiding van het jubileum.
+Ik vind de stekker niet.
+Pesticiden kunnen giftig zijn.
+Kan jij figuren zien in de wolken?
+Ik heb mijn vettige handen met zijn zeep gewassen.
+Hij teelde al jarenlang dezelfde gewassen.
+De butler serveerde het eten. 
+Hun bedrijf bevindt zich in een grijze zone.
+Niemand weet goed wat ze gaan doen met de weeskinderen.
+Er is een wet gestemd vanmorgen.
+Zij zetelt in het parlement.
+De regering is met spoed samengekomen. 
+Hij had argwaan en besloot om het contract niet te ondertekenen.
+Mag ik je handtekening?
+Voor hun huis zijn ze een lening van twintig jaar aangegaan.
+Het boek was vlot geschreven.
+Zij kon altijd goede verhalen vertellen.
+Het was een spannende achtervolging.
+Ik ben tegen de doodstraf.
+Zij is haar land moeten ontvluchten omdat de vervolgt werd voor politieke misdaden. 
```

Comment:
"is .. moeten ontvluchten" => "moest .. ontvluchten"/"is .. ontvlucht"
"de" => "ze"
"vervolgt" => "vervolgd"